includes/class-sensei.php

```diff
@@ -757,8 +762,10 @@ public function check_user_permissions( $page = '' ) {
 				// Check for WC purchase
 				$lesson_course_id = get_post_meta( $post->ID, '_lesson_course', true );
 
-				$update_course = Sensei_WC::course_update( $lesson_course_id );
-				$is_preview    = Sensei_Utils::is_preview_lesson( $post->ID );
+				if ( method_exists( 'Sensei_WC', 'course_update' ) ) {
+					Sensei_WC::course_update( $post->ID );
```

Comment:
Should `$post->ID` be `$lesson_course_id` here?

Reply:
9ee50332b4693a93209ab726fbadb5ee3dcd146a